pkg/controller/constants.go

```diff
@@ -19,11 +19,22 @@ const (
 	// successful reconciliation by the controller.
 	controllerProcessedAnnotation = "operator.openshift.io/external-secrets-processed"
 
+	// certManagerInjectCAFromAnnotation is the annotation added to external-secrets resource once
+	// if certManager field is enabled in webhook config
+	// after successful reconciliation by the controller.
+	certManagerInjectCAFromAnnotation = "cert-manager.io/inject-ca-from"
+
 	externalsecretsObjectName = "cluster"
 
 	// certificateCRDGroupVersion is the group and version of the Certificate CRD provided by cert-manager project.
 	certificateCRDGroupVersion = "cert-manager.io/v1"
 
 	// certificateCRDName is the name of the Certificate CRD provided by cert-manager project.
 	certificateCRDName = "certificates"
+
+	externalSecretsWebhook = "external-secrets/external-secrets-webhook"
+)
+
+var (
```

Comment:
Could we have this in `pkg/controller/validatingwebhook.go`, since its specific for webhook?